source/validator/contracts/Validator.sol

```diff
@@ -533,9 +534,14 @@ contract Validator {
       address owner = IERC721(party.token).ownerOf(party.id);
       return (owner == party.wallet);
     }
-
-    uint256 balance = IERC20(party.token).balanceOf(party.wallet);
-    return (balance >= party.amount);
+    else if (party.kind == ERC1155_INTERFACE_ID) {
```

Comment:
generalize line 526: for ERC721, CryptoKitties, and ERC20 tokens => supported token types